_posts/2018-08-02-pulp-2.16.4-generally-available.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Pulp 2.16.4 Generally Available
+author: David Davis
+tags:
+  - release
+---
+
+Pulp 2.16.4 is now Generally Available in the stable repositories:
+
+* [pulp-2-stable](https://repos.fedorapeople.org/pulp/pulp/stable/2/)
+* [pulp-stable](https://repos.fedorapeople.org/pulp/pulp/stable/latest/)
+
+This release includes bug fixes for Pulp, Debian Support, Python Support, and RPM Support.
+
+## Upgrading
+
+The Pulp 2 stable repositories are included in the pulp repo files:
+
+- [Fedora](https://repos.fedorapeople.org/repos/pulp/pulp/fedora-pulp.repo)
+- [RHEL 7](https://repos.fedorapeople.org/repos/pulp/pulp/rhel-pulp.repo)
+
+After enabling the pulp-stable or pulp-2-stable repository, you'll want to follow the standard
+upgrade path with migrations:
+
+```sh
+$ sudo systemctl stop httpd pulp_workers pulp_resource_manager pulp_celerybeat pulp_streamer goferd
+$ sudo yum upgrade
+$ sudo -u apache pulp-manage-db
+$ sudo systemctl start httpd pulp_workers pulp_resource_manager pulp_celerybeat pulp_streamer goferd
+```
+
+The `pulp_streamer` and `goferd` services should be omitted if those services are not installed.
+
+
+## Issues Addressed
+
+### Pulp
+- 3828	pulp-gen-ca-certificate script fails when /proc/sys/crypto/fips_enabled doesn't exist
```

Comment:
It would be cool if the issue number was linkable or something like that. It's ok if you want to merge w/o changing this.